Gemfile

```diff
@@ -11,6 +11,10 @@ gem 'rubyhorn', git: "https://github.com/avalonmediasystem/rubyhorn.git"
 gem 'shingoncoder'
 gem 'zencoder'
 
+group :development, :test do
+  gem 'bixby', git: "https://github.com/samvera-labs/bixby.git", branch: "rubocop1"
+end
```

Comment:
I guess this is to switch to a non-released branch of bixby? Why is this necessary as part of the bump to version of ruby/rails tested? 

Can you put some source comments in above this code, explaining why a custom branch of bixby was required, and under what circumstances we could go back to main bixby?

I am worried a custom branch of bixby (that may not ever be updated) is going to get us stuck with a certain version of rubocop rules forever. I ran into trouble trying to write code based on a very old rubycop rules just now, that also when I upgraded bixby/rubocop made me aware of the "deferred maintenance" of staying on a frozen-in-time bixby/rubocop, which leads to my concern.

Reply:
I would like to clear this up too.  The rubocop1 branch is an active PR that upgrades bixby to rubocop 1.  If that gets merged and released then I'd remove this.  So I view this as temporary and might expect to leave this PR open until that gets resolved assuming it is soon.

Reply:
If it were me, I'd keep this repo on the latest actual release of bixby, and not upgrade to rubocop 1 until there's a bixby release for it. Becuase of the risk this thing ends up forgotten about again and stays on a bixby branch forever and that means more technical debt and headache for someone else down the line. And in general my aversion to using unreleased branches unless absolutely necessary, for such reasons -- and why is it necessary here to go past the latest bixby release with it's rubocop?

But this is your baby here, I give this PR my peer approval for whatever you want to do.

Reply:
I've updated this PR to pull in the released bixby 4.0.0 which supports rubocop 1.x instead of the branch.